### PR TITLE
Slightly adjust match/multiplayer designs

### PR DESCRIPTION
**Pull request by smoogipoo**
From:
Screenshot_2020-06-25_22-26-14
Screenshot_2020-06-25_22-08-19

To:
Screenshot_2020-06-25_22-29-21
Screenshot_2020-06-25_22-22-04

### CODE REVIEW

**Comment by peppy** [comment]
How does this look in vertical aspect ratio?

**Comment by peppy** [comment]
Also just a personal opinion, but the match screen feels messy and overwhelming right now. Too much displayed and very hard to know what is related to / doing what.

I'll probably take some time to come up with a basic design in the near future, since it's clear the current "ask designer for everything" doesn't scale well (requirements are hard to convey when the designer isn't actually using / hasn't historically used the systems).

**Review by peppy — CHANGES_REQUESTED** [review]

File: osu.Game/Tests/Visual/ScreenTestScene.cs [hunk 1]
```diff
@@ -33,8 +33,8 @@ protected ScreenTestScene()
         [SetUpSteps]
         public virtual void SetUpSteps() => addExitAllScreensStep();
 
-        [TearDownSteps]
-        public virtual void TearDownSteps() => addExitAllScreensStep();
+        // [TearDownSteps]
```
[the remark this answers is not in the document]
Comment (smoogipoo) [reply osu.Game/Tests/Visual/ScreenTestScene.cs:36]: Whoops I didn’t mean to commit this.

**Comment by smoogipoo** [comment]
Going to close this one for now as we rethink the layout altogether.

**Comment by peppy** [comment]
Reopening to use as a starting point for getting the match screen into a better state. Structure wise, I believe we still want many of the elements added in this. May revert some pieces as I go.

**Review by peppy — APPROVED** [review]

**Pull request merged**